ydb/public/sdk/cpp/client/ydb_query/client.cpp

```diff
@@ -18,7 +18,8 @@
 
 namespace NYdb::NQuery {
 
-using TRetryContextAsync = NRetry::Async::TRetryContext<TQueryClient, TAsyncExecuteQueryResult>;
+using TRetryContextResultAsync = NRetry::Async::TRetryContext<TQueryClient, TAsyncExecuteQueryResult>;
+using TRetryContextStatusAsync = NRetry::Async::TRetryContext<TQueryClient, TAsyncStatus>;
```

Comment:
```suggestion
using TRetryContextAsync = NRetry::Async::TRetryContext<TQueryClient, TAsyncStatus>;
```

Тоже не имеет смысл писать StatusAsync